xds/src/test/java/io/grpc/xds/XdsNameResolverTest.java

```diff
@@ -119,6 +119,7 @@ XdsChannel createChannel(List<ServerInfo> servers) throws XdsInitializationExcep
 
   @Before
   public void setUp() {
+    XdsNameResolver.enableTimeout = true;
```

Comment:
Turn the flag back to the original value in `tearDown()`. I don't know whether other test classes will run in the same JVM in some settings.

Reply:
We do not test the behavior when this env variable is unset.

This env variable is just used to exclude one or two lines of logic for emitting a service config, we shouldn't make it involve too complicate code change, as it should be easy to be removed later. Since the gating logic is super clear, we shouldn't bother verify the outcome of this gate, without complicate things when for its removal.